nextcord/abc.py

```diff
@@ -684,6 +684,13 @@ def permissions_for(self, obj: Union[Member, Role], /) -> Permissions:
             denied = Permissions.all_channel()
             base.value &= ~denied.value
 
+        # if you are timed out then you lose all permissions except view_channel and read_message_history
+        if obj.communication_disabled_until is not None:
+            allowed = Permissions.none()
+            allowed.view_channel = True
+            allowed.read_message_history = True
```

Comment:
Looks good :+1: I might suggest making this more concise

```suggestion
            allowed = Permissions(view_channel=True, read_message_history=True)
```